### ISSUE TITLE
site not accessible for screen readers with mobile devices completely

### ISSUE THREAD

**Issue by jlvp1998** [vorple/borogove-ide#29]
so I've noticed whenever I went to borogove.app/inform7 and I start typing in the code to make a game and then press go. I can't download the file or anything after it completes however, on a computer, I can tap on the clickable thing near the text field and it shows it with like settings and other things there the problem is this is not accessible completely. This should be accessible with a screen reader, and I hope it can be even the clickable thing isn't labeled if you're using it on Windows and there should be a label there as well.

**Comment by jlvp1998** [comment]
OK, so to update you on my latest issue here I've noticed that majority of each Software or each project I create with each type of interactive fiction language. It does not support screen readers mainly for the phone but one thing I've noticed is Zell does not support a mobile screen reader at all because it doesn't show me any text field or anything so I can't do anything with that now on Windows I believe these are supported with a screen reader again the button needs to be labeled to where you can go into your settings and compile it and all that good stuff, however, on the mobile browsers that button or that clickable thing isn't even shown
